src/test/scala/com/lightbend/kafkalagexporter/ConsumerGroupCollectorSpec.scala

```diff
@@ -126,6 +130,48 @@ class ConsumerGroupCollectorSpec extends FreeSpec with Matchers with TestData wi
     }
   }
 
+  "ConsumerGroupCollector should sum the group offset lag metrics by topic and send" - {
+    val reporter = TestInbox[MetricsSink.Message]()
+
```

Comment:
If the intention is to have sum lag per group and topic, can you add a test case where the group has more than 1 topic?

Reply:
Yes! Added this.